tfx/components/example_validator/component_test.py

```diff
@@ -30,13 +30,18 @@ class ExampleValidatorTest(tf.test.TestCase):
   def testConstruct(self):
     statistics_artifact = standard_artifacts.ExampleStatistics()
     statistics_artifact.split_names = artifact_utils.encode_split_names(
-        ['eval'])
+        ['train', 'eval'])
+    exclude_splits = ['eval']
     example_validator = component.ExampleValidator(
         statistics=channel_utils.as_channel([statistics_artifact]),
         schema=channel_utils.as_channel([standard_artifacts.Schema()]),
-    )
+        exclude_splits=exclude_splits)
     self.assertEqual(standard_artifacts.ExampleAnomalies.TYPE_NAME,
                      example_validator.outputs['anomalies'].type_name)
+    self.assertEqual('["train"]', artifact_utils.get_single_instance(
+        list(example_validator.outputs['anomalies'].get())).split_names)
+    self.assertEqual(example_validator.spec.exec_properties['exclude_splits'],
+                     ['eval'])
```

Comment:
exec_properties['exclude_splits'] doesn't allow list type, we might need to convert it to json str